samples/subsys/ipc/ipm_psoc6/client/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.8.2)
+#
+# Copyright (c) 2018, Cypress Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(NONE)
+
+target_include_directories(app PRIVATE $ENV{ZEPHYR_BASE}/drivers)
+target_sources(app PRIVATE src/main_client.c)
```

Comment:
Is it possible to build both the client and server using one command, like demonstrated in samples/subsys/ipc/ipm_mcux/CMakeLists.txt? Ignore this comment if not.